### DISCUSSION TITLE
Ian Wold | Just Use PostgreSQL

### DISCUSSION THREAD

**Discussion by giscus** [IanWold/ianwold.github.io#12] [Posts]
# Ian Wold | Just Use PostgreSQL

Ian Wold is a Software Engineer, Architect, and Team Leader in Minneapolis.

https://ian.wold.guru/Posts/just_use_postgresql.html

**Comment by IanWold** [comment]
One note I can make here regarding a compelling case to not use PostgreSQL: I recently made a [real-time SignalR app](https://ian.wold.guru/Posts/free_planning_poker.html) which doesn't require long-term persistence. SignalR is able to use Redis as a backbone, and the only persistence I need is to be able to get incoming users up-to-speed. Redis made sense in this case, I think.

**Comment by IanWold** [comment]
Oh look at that I have a bunch of resources linked on a recent book club post: https://ian.wold.guru/Posts/book_club_6-7-2024.html

**Comment by andrewwillette** [comment]
Postgres es la verdad!!!
